source/shared/cpp/AdaptiveCardsSharedModel/AdaptiveCardsSharedModelUnitTest/CustomParsingForIOSTest.cpp

```diff
@@ -0,0 +1,116 @@
+#include "stdafx.h"
+#include "CppUnitTest.h"
+#include "TextBlock.h"
+#include <time.h>
+#include <Windows.h>
+#include <StrSafe.h>
+#include "SharedAdaptiveCard.h"
+#include "BaseCardElement.h"
+#include "CustomObjectDelegateForIOS.h"
+
+using namespace Microsoft::VisualStudio::CppUnitTestFramework;
+using namespace AdaptiveCards;
+using namespace std;
+
+namespace AdaptiveCardsSharedModelUnitTest
+{
+    TEST_CLASS(CustomParsingForIOSTest)
+    {
+    public:
+        TEST_METHOD(CanGetCustomJsonPayload)
+        {
+            std::string testJsonString = 
+            "{\
```

Comment:
nit - you may use `R"JSON(<string literal>)JSON"` to skip escaping special character, if we compile with C++ 11. See http://en.cppreference.com/w/cpp/language/string_literal

Reply:
thank you.

---
In reply to: [172231603](https://github.com/Microsoft/AdaptiveCards/pull/1211#discussion_r172231603) [](ancestors = 172231603)